eo-maven-plugin/src/main/java/org/eolang/maven/BinarizeMojo.java

```diff
@@ -75,6 +75,19 @@ public final class BinarizeMojo extends SafeMojo {
     @SuppressWarnings("PMD.UnusedPrivateField")
     private File generatedDir;
 
+    /**
+     * The directory with eo_env rust project. It is a necessary dependency
+     * that provides rust-eo interaction.
+     * @checkstyle MemberNameCheck (8 lines)
+     */
+    @Parameter(
+        property = "eo.eo_env",
```

Comment:
@levBagryansky in Java world underscores are not used, we prefer CamelCase instead.